Mol_Analysis.py

```diff
@@ -615,7 +615,13 @@ def plotall(self, local=True, ncols=3, nrows=2, **kwargs):
             sys.exit("More than 6 plot! Please use -large")
         print("Plotting %s plots" % (nplots))
         self.fig.suptitle(self.odir)
-        self.fig.add_subplot(ncols, nrows, 1)
+        #self.fig.add_subplot(ncols, nrows, 1)
+        
+        protein_name = self.odir.split('/')[6]  
```

Comment:
why the 6th element of the path, specifically? This doesn't seem very general (what happens if the code is run in a folder with a completely different path)?

I think this should be set in the configuration file instead